### PR TITLE
[JS] Topic : Objects

### LINKED ISSUE #246: JS: Objects
# Objects
Objects in JavaScript.

## Directory
```
Topics/5.Syntax/6.Functions_And_Scope
```

## Description
* About JavaScript objects.
* How to define them?
* Getters and setters.
* What are keys and values?

### PR DESCRIPTION
**Pull request by AbhinandanAdhikari**
<hr>

## Description 📜

Created a tutorial for JavaScript Objects. 
Fixes #246

<hr>

## Type of change 📝

- [ ] Audio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev (Flutter, Java, Kotlin)
- [ ] Backend Dev (Java, .NET, Python)
- [ ] C/CPP
- [ ] Competitive Programming
- [ ] Cyber Security
- [ ] DSA
- [ ] Database
- [ ] Datascience with Python
- [ ] Datascience with R
- [ ] Frontend Dev (HTML, CSS, JS)
- [ ] Frontend Dev (React, Angular, Vue)
- [ ] Golang
- [ ] Interview Prep
- [ ] Java
- [x] JavaScript
- [ ] MERN
- [ ] Machine Learning
- [ ] Open Source
- [ ] Python
- [ ] Rust
- [ ] Statistics
- [ ] UI/UX

<hr>
 
## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by AbhinandanAdhikari** [comment]
@paramsiddharth Sir, I've made the new Pull Request, please review it.

**Comment by paramsiddharth** [comment]
@dakshsethi @Creatoon Please review.

**Comment by AbhinandanAdhikari** [comment]
@dakshsethi @Creatoon Sir, please review it

**Review by paramsiddharth — APPROVED** [review]

**Comment by AbhinandanAdhikari** [comment]
@dakshsethi @Creatoon @s4-2 Mentors! Please review it.

**Comment by s4-2** [comment]
@AbhinandanAdhikari Well Done... It looks good for me.
Please @Creatoon review it once.

File: Web_Development/Javascript/Topics/5.Syntax/Objects/README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+The above examples uses ```nation``` to set the value of ```nationality``` property.
+
+# Difference between Getters and Setters
+The basic difference between them is that __Getters__ are the object accessor that is used to get the property from an object, whereas __Setters__ are the object accessor that is used to set the property of an object.
+
+# Conclusion
+
+Congratulations! You've successfully learnt about JavaScript Object.
+
+Keeping exploring JavaScript :wave:
```
Comment (s4-2) [inline Web_Development/Javascript/Topics/5.Syntax/Objects/README.md:134]: Little grammatiacal correction is require **Keep Exploring JavaScript**. Hope you get it:-)

File: Web_Development/Javascript/Topics/5.Syntax/Objects/README.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
+# JavaScript Objects
+An JavaScript object is anything(entity) having properties and methods.
```
Comment (Creatoon) [inline Web_Development/Javascript/Topics/5.Syntax/Objects/README.md:2]: Article error -> A JavaScript

File: Web_Development/Javascript/Topics/5.Syntax/Objects/README.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+JavaScript is an object-based language, almost everything is an object. For eg:- numbers, strings etc are objects if they are defined with the ```new``` keyword.
```
Comment (Creatoon) [inline Web_Development/Javascript/Topics/5.Syntax/Objects/README.md:4]: object oriented will be more effective than object based

File: Web_Development/Javascript/Topics/5.Syntax/Objects/README.md (hunk 1 above)
Comment (AbhinandanAdhikari) [reply Web_Development/Javascript/Topics/5.Syntax/Objects/README.md]: Ma'am, I've corrected it.

**Review by Creatoon — CHANGES_REQUESTED** [review]

File: Web_Development/Javascript/Topics/5.Syntax/Objects/README.md (hunk 3 above)
Comment (s4-2) [reply Web_Development/Javascript/Topics/5.Syntax/Objects/README.md:4]: @AbhinandanAdhikari Yes, well done your PRs look good from my side. Wait till others mentor approve it:)

Comment (AbhinandanAdhikari) [reply Web_Development/Javascript/Topics/5.Syntax/Objects/README.md]: @s4-2 @Creatoon @dakshsethi Mentors! Made the changes, please review it.

**Review by Creatoon — APPROVED** [review]

**Comment by AbhinandanAdhikari** [comment]
@dakshsethi @paramsiddharth  @s4-2 Please review the changes made.

**Review by dakshsethi — APPROVED** [review]
Looks good to me!

**Comment by AbhinandanAdhikari** [comment]
@paramsiddharth Sir, since it is approved by all mentors, will this PR get merged?

**Comment by AbhinandanAdhikari** [comment]
@s4-2 @paramsiddharth Please review this Pull Request

**Comment by dakshsethi** [comment]
> @paramsiddharth Sir, since it is approved by all mentors, will this PR get merged?

Yes @AbhinandanAdhikari it will surely get merged. Let @paramsiddharth and @s4-2 approve this.

**Pull request merged**